### PR TITLE
free container disk

### PR DESCRIPTION
**Pull request by panbingkun**
The pr aims to `free container disk`.

### CODE REVIEW

**Comment by panbingkun** [comment]
cc @yaooqinn

**Review by yaooqinn — APPROVED** [review]

File: .github/workflows/doc_gen.yml [hunk 1]
```diff
@@ -10,11 +10,42 @@ on:
 
 jobs:
   lint:
-    name: check wether all documentation was generated with the right Jekyll version
+    name: check whether all documentation was generated with the right Jekyll version
```
Comment (panbingkun) [inline .github/workflows/doc_gen.yml:13]: fix typo

**Comment by srowen** [comment]
Looks good as a stopgap fix.

**Review by dongjoon-hyun — COMMENTED** [review]
Thank you!

**Comment by panbingkun** [comment]
> Looks good as a stopgap fix.

Yea, a temporary solution. 
As we release more and more version documents, the space of this repo has become larger and larger. 
Perhaps we can split some historical version documents (such as `0. x`, `1. x`, `2. x`) and process them as `cold` data documents.

**Pull request merged**